src/apps/integrations/tests/test_domain.py

```diff
@@ -0,0 +1,19 @@
+import pytest
```

Comment:
In the project unit tests are stored under "tests/unit" subfolder, integration under "tests". Please follow this pattern (see e.g. apps/activities/tests)